packages/api-fetch/src/middlewares/preloading.js

```diff
@@ -72,11 +72,11 @@ function createPreloadingMiddleware( preloadedData ) {
 				cache[ method ] &&
 				cache[ method ][ path ]
 			) {
-				return Promise.resolve(
-					parse
-						? cache[ method ][ path ].body
-						: cache[ method ][ path ]
-				);
+				// Unsetting the cache key ensures that the data is only preloaded a single time
+				const cacheData = cache[ method ][ path ];
+				delete cache[ method ][ path ];
```

Comment:
Tiny nit: would be nice to format this similarly to the above branch

```suggestion
				const cacheData = cache[ method ][ path ];
				
				// Unsetting the cache key ensures that the data is only preloaded a single time
				delete cache[ method ][ path ];
				
```

Reply:
I agree. 
Fixed in d71c48f4cf4c51da33a598e5381f21cd07d8eab9